cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -399,6 +399,7 @@ func (oi *OVFImporter) setUpImportWorkflow() (*daisy.Workflow, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error parsing workflow %q: %v", ovfImportWorkflow, err)
 	}
+	workflow.ForceCleanupOnError = true
```

Comment:
Changing behavior outside of workflow is not the best. Why don't we configure it in ovf .wf file?

Reply:
Non-NoCleanup disks will be cleaned up automatically without having to force anything. I'm not sure what your new flag would do.

Reply:
We already heavily modify OVF import workflow in the wrapper, so much so that import would not work in most cases if it was run directly with Daisy (e.g. multi disks would not be imported). Since no one will ever use OVF workflow directly, it's more of a question of style than correctness where to put this change. Having it in the wrapper would allow us to make it configurable via a wrapper flag in the future, so I'd rather keep it as is.

Reply:
Hi Zoran, you are right that Non-NoCleanup disks will be Auto cleaned up on error. However, they won't be cleaned up on unexpected exit (ctrl+c, for example). My change will try to handle that. But I realized that doesn't need a flag.
I still prefer to have a clearer name because it is confusion, it made me feel cleanup won't happen without this flag. Maybe "DeleteNoCleanupArtifactsOnError" or something

Reply:
I feel "forced cleanup" describes "Delete NoCleanup artifacts" pretty well and is more concise. If the flag was named "CleanupOnError" then I would understand how it would be confusing. "Forced" is pretty explicit that it's not regular cleanup. There's documentation to resolve any remaining confusion.

Reply:
OK. Ideally we can have something like NoCleanup=Always/OnlyOnSuccess/false to avoid this, but for backward compatible we have to do it like this.